wagtail/documents/views/bulk_actions/delete.py

```diff
@@ -0,0 +1,37 @@
+from django.utils.translation import gettext_lazy as _
+from django.utils.translation import ngettext
+
+from wagtail.core import hooks
+from wagtail.documents.permissions import permission_policy
+from wagtail.documents.views.bulk_actions.document_bulk_action import DocumentBulkAction
+
+
+class DeleteBulkAction(DocumentBulkAction):
+    display_name = _("Delete")
+    action_type = "delete"
+    aria_label = _("Delete documents")
+    template_name = "wagtaildocs/bulk_actions/confirm_bulk_delete.html"
+    action_priority = 10
+    classes = {'serious'}
+
+    def check_perm(self, document):
+        return permission_policy.user_has_permission_for_instance(self.request.user, 'delete', document)
+
+    def execute_action(cls, documents):
```

Comment:
Why is this `cls` rather than `self`? This doesn't seem to be a classmethod.

Reply:
Actually it is a classmethod. It is used by another classmethod `execute` which is used to perform bulk actions using a management command. This can be changed to a simple method though.

Reply:
Should it not get the classmethod decorator in that case?

Reply:
Yes, I'll add that